test/functional/wallet_bumpfee.py

```diff
@@ -17,7 +17,7 @@
 import io
 
 from test_framework.blocktools import add_witness_commitment, create_block, create_coinbase, send_to_witness
-from test_framework.messages import BIP125_SEQUENCE_NUMBER, CTransaction
+from test_framework.messages import BIP125_SEQUENCE_NUMBER, CTransaction, FromHex
```

Comment:
Remove this new import of `FromHex` to satisfy the linter.

Reply:
fixed.